core/arch/arm/arm.mk

```diff
@@ -245,6 +245,8 @@ ta-mk-file-export-add-ta_arm32 += CROSS_COMPILE32 ?= $$(CROSS_COMPILE)_nl_
 ta-mk-file-export-add-ta_arm32 += CROSS_COMPILE_ta_arm32 ?= $$(CROSS_COMPILE32)_nl_
 ta-mk-file-export-add-ta_arm32 += COMPILER ?= gcc_nl_
 ta-mk-file-export-add-ta_arm32 += COMPILER_ta_arm32 ?= $$(COMPILER)_nl_
+
+ta-mk-file-export-add-ta_arm32 += PYTHON3 ?= $$(PYTHON3)
```

Comment:
You need to append `_nl_` (turned into a new line character when `ta-mk-file-export-add-ta_arm32` is processed).

Reply:
Fixed